packages/contractkit/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import Web3 from 'web3'
+
+export { Address, CeloContract } from 'src/base'
+export * from 'src/kit'
+export { newAttestations } from './generated/Attestations'
+export { newBondedDeposits } from './generated/BondedDeposits'
+export { newEscrow } from './generated/Escrow'
+export { newExchange } from './generated/Exchange'
+export { newGasCurrencyWhitelist } from './generated/GasCurrencyWhitelist'
+export { newGasPriceMinimum } from './generated/GasPriceMinimum'
+export { newGoldToken } from './generated/GoldToken'
+export { newGovernance } from './generated/Governance'
+export { newMultiSig } from './generated/MultiSig'
+export { newRandom } from './generated/Random'
+export { newRegistry } from './generated/Registry'
+export { newReserve } from './generated/Reserve'
+export { newSortedOracles } from './generated/SortedOracles'
+export { newStableToken } from './generated/StableToken'
+export { newValidators } from './generated/Validators'
+
+export function newWeb3(url: string) {
```

Comment:
Is this function necessary?

Reply:
not now... but if we end up wrapping web3 we'll probably want it